### PR TITLE
Slim down ehr tests

### PR DESCRIPTION
**Pull request by stefpiatek**
Reduce use of docker containers
and run tests in local pytest

### CODE REVIEW

**Review by jeremyestein — APPROVED** [review]
I'm going to merge this now as there's no need to have two branches here. My comments are for our reference. I will fix the easy ones.

File: pixl_ehr/tests/docker-compose.yml [hunk 1]
```diff
@@ -15,49 +15,6 @@ version: "3.8"
...
-      EMAP_UDS_NAME: emap  # database name
-      EMAP_UDS_USER: postgres
-      EMAP_UDS_PASSWORD: postgres
-      EMAP_UDS_SCHEMA_NAME: star
-      COGSTACK_REDACT_URL: http://cogstack-api:8000/redact
-    ports:
-      - "9000:8000"
-      - "443:443"
-
   queue:
```
[the remark this answers is not in the document]
Comment (stefpiatek) [reply pixl_ehr/tests/docker-compose.yml:18]: Long term most probably shouldn't but these were low hanging fruit

File: pixl_ehr/tests/conftest.py [hunk 2]
```diff
@@ -15,6 +15,22 @@
...
+os.environ["PIXL_DB_USER"] = "postgres"
+os.environ["PIXL_DB_PASSWORD"] = "postgres"
+os.environ["PIXL_DB_EHR_SCHEMA_NAME"] = "emap_data"
+os.environ["EMAP_UDS_HOST"] = "localhost"
+os.environ["EMAP_UDS_PORT"] = "35433"
+os.environ["EMAP_UDS_NAME"] = "emap"
+os.environ["EMAP_UDS_USER"] = "postgres"
+os.environ["EMAP_UDS_PASSWORD"] = "postgres"
+os.environ["EMAP_UDS_SCHEMA_NAME"] = "star"
+os.environ["COGSTACK_REDACT_URL"] = "test"
```
[the remark this answers is not in the document]
Comment (stefpiatek) [reply pixl_ehr/tests/conftest.py]: yeah required environmental variable, even if we're mocking out the requests

File: pixl_ehr/pyproject.toml [hunk 3]
```diff
@@ -42,7 +42,7 @@ markers = ["processing"]
 extend = "../ruff.toml"
 
 [tool.ruff.extend-per-file-ignores]
-"./tests/**" = ["D1"]
+"./tests/**" = ["D1", "S105"]
```
[the remark this answers is not in the document]
Comment (stefpiatek) [reply pixl_ehr/pyproject.toml]: I think hardcoding in tests is fine?

File: pixl_ehr/src/pixl_ehr/_databases.py [hunk 4]
```diff
@@ -33,14 +33,17 @@
 class Database:
     """Fake database wrapper"""
```
[the remark this answers is not in the document]
Comment (stefpiatek) [reply pixl_ehr/src/pixl_ehr/_databases.py:34]: Yeah we should change this

**Pull request merged**